pkg/cluster/cluster.go

```diff
@@ -212,8 +212,6 @@ func (c *Cluster) send(ev *clusterEvent) {
 	select {
 	case c.eventCh <- ev:
```

Comment:
We should check the length of channel and have a watermark warning. This will help us have better insights into how operator runs.